### PR TITLE
Downgrade to rmwc 4.0.6 to support IE

### PR DESCRIPTION
**Pull request by biovisualize**
This PR is for downgrading the RMWC version used by omnisci-ui to the previous one ([4.0.6](https://rmwc.io/version/4.0.6/)) to make it compatible with IE. The only piece that really needs to work is the dialog since we will use it in 2 projects for this release.

https://omnisci.github.io/omnisci-ui/styleguide/

RMWC works with typescript, although I merely silenced it with `any` and such until we find a solution for supporting this one client using IE11 and we can upgrade again. It required some minimal change in API and css needed to be adjusted to use an older version of the official material css. I estimate that upgrading it back to latest will not be a huge effort. And I did it twice by now.

Once approved, I will manually merge this branch (probably using a `reset --hard`) as it's expected to not be compatible/mergeable in master.

Closes https://jira.omnisci.com/browse/FE-8600

### CODE REVIEW

File: src/components/dialog/Dialog.md [hunk 1]
```diff
@@ -147,7 +151,7 @@ initialState = {
         unelevated
         ripple={false}
         className="danger"
-        action="accept"
+        action="agree"
```
Comment (jrajav) [inline src/components/dialog/Dialog.md]: Was this change intentional?

Comment (biovisualize) [reply src/components/dialog/Dialog.md]: I was just testing that the action string is what is internally routed as an argument for onClose. It's not significant though so I will clean that out.

**Comment by biovisualize** [comment]
I verified that it works on IE: https://s.codepen.io/biovisualize/debug/XLNeoO/VJrxxVDEnLar

It needs babel-polyfill and material-icons, which should already be available in Immerse.

Capture3

**Comment by biovisualize** [comment]
This branch was merged manually.
